jina/serve/runtimes/request_handlers/data_request_handler.py

```diff
@@ -152,6 +152,7 @@ async def handle(self, requests: List['DataRequest']) -> DataRequest:
         )
 
         # executor logic
+        # somehow exception not raised here and skips directly to counter, should be caught in jina/serve/runtimes/worker/__init__.py instead
```

Comment:
I am not sure if this is a useful comment outside of the context of reviewing this PR. A 3rd party will not know what exception and what counter this is talking about.